pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -334,6 +334,10 @@ def test_setitem_complete_column_with_array(self):
                 "d": [1, 1, 1],
             }
         )
+        expected["c"] = expected["c"].astype(arr.dtype)
+        expected["d"] = expected["d"].astype(arr.dtype)
+        assert expected["c"].dtype == arr.dtype
+        assert expected["d"].dtype == arr.dtype
```

Comment:
What changed here? (I wouldn't expect the above test to change behaviour related to this PR, as it's adding new columns)

Reply:
IIRC this was needed for the 32bit builds